src/pages/(study)/reflection/ReflectionDetailPage.tsx

```diff
@@ -0,0 +1,214 @@
+import { useState, useEffect } from 'react';
+import { useNavigate, useParams } from 'react-router-dom';
+import { ROUTES, ROUTE_BUILDERS, ROUTE_PARAMS } from '@/constants';
+import { REFLECTION_TEXTS, SCORE_LABELS, SCORE_RANGE } from './constants';
+import { mockSchedules, mockReflectionDetails } from './mock';
+import { ScoreSlider, ScheduleDropdown } from './components';
+import type { ReflectionFormData, Schedule } from './types';
+
+/**
+ * 회고 작성/수정 페이지
+ */
+const ReflectionDetailPage = () => {
+  const navigate = useNavigate();
+  const { [ROUTE_PARAMS.reflectionId]: reflection_id, study_id } = useParams();
+  const isEdit =
+    Boolean(reflection_id) && !window.location.pathname.endsWith('/write');
```

Comment:
```suggestion
    Boolean(reflection_id)
```

이렇게 reflection_id 값의 존재 유무만으로 판단할 수는 없을까요?
`/write` 경로를 하드코딩하는건 안전한 방법은 아닌 것 같아서요.

Reply:
window.location.pathname.endsWith('/write') 하드코딩 부분을 제거하고 reflection_id 존재 여부만으로 판단하도록 수정하였습니다!